bigbluebutton-html5/imports/ui/components/audio/container.jsx

```diff
@@ -1,32 +1,41 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
 import { withModalMounter } from '/imports/ui/components/modal/service';
-
+import PropTypes from 'prop-types';
 import Service from './service';
 import Audio from './component';
 import AudioModal from './audio-modal/component';
 
-class AudioContainer extends Component {
-  render() {
-    return (
-      <Audio {...this.props}>
-        {this.props.children}
-      </Audio>
-    );
-  }
-}
+const propTypes = {
+  children: PropTypes.element,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const AudioContainer = props =>
+  (<Audio {...props}>
+    {props.children}
+  </Audio>
+  );
 
 let didMountedAutoJoin = false;
```

Comment:
We should probably rename this variable, `didMountedAutoJoin` can probably be just `mountedAutoJoin`